.github/workflows/test-target.yml

```diff
@@ -102,18 +102,25 @@ jobs:
       DDEV_E2E_AGENT_PY2: "${{ inputs.platform == 'windows' && (inputs.agent-image-windows-py2 || 'datadog/agent-dev:master-py2-win-servercore') || inputs.agent-image-py2 }}"
       # Test results for later processing
       TEST_RESULTS_BASE_DIR: "test-results"
-      TEST_RESULTS_DIR: "test-results/${{ inputs.job-name }}"
       # Tracing to monitor our test suite
       DD_ENV: "ci"
       DD_SERVICE: "ddev-integrations-${{ inputs.repo }}"
       DD_TAGS: "team:agent-integrations,platform:${{ inputs.platform }},integration:${{ inputs.target }}"
       DD_TRACE_ANALYTICS_ENABLED: "true"
       # Capture traces for a separate job to do the submission
-      TRACE_CAPTURE_BASE_DIR: ".trace-captures"
-      TRACE_CAPTURE_FILE: ".trace-captures/${{ inputs.job-name }}"
-      TRACE_CAPTURE_LOG: ".trace-captures/output.log"
+      TRACE_CAPTURE_BASE_DIR: "trace-captures"
+      TRACE_CAPTURE_LOG: "trace-captures/output.log"
 
     steps:
+
+    - name: Set environment variables with sanitized paths
+      run: |
+        # We want to replace leading dots as they will make directories hidden, which will cause them to be ignored by upload-artifact and EnricoMi/publish-unit-test-result-action
+        JOB_NAME=$(echo "${{ inputs.job-name }}" | sed 's/^\./Dot/')
+
+        echo "TEST_RESULTS_DIR=$TEST_RESULTS_BASE_DIR/$JOB_NAME" >> $GITHUB_ENV
+        echo "TRACE_CAPTURE_FILE=$TRACE_CAPTURE_BASE_DIR/$JOB_NAME" >> $GITHUB_ENV
```

Comment:
This is line actually not entirely needed because TRACE_CAPTURE_FILE is a file, not a directory. We then provide the exact file name to upload-artifact. This makes upload-artifact find the file despite it being hidden, unlike when globs are used. But it's pretty fragile so it's probably a good idea to just get rid of any hidden file or directory in case someone modifies this in the future.